shell/edit/fleet.cattle.io.gitrepo.vue

```diff
@@ -413,8 +413,9 @@ export default {
           type:     SECRET,
           metadata: {
             namespace:    this.value.metadata.namespace,
-            generateName: 'auth-'
-          },
+            generateName: 'auth-',
+            labels:       { 'fleet.cattle.io/managed': 'true' }
```

Comment:
This should reference https://github.com/rancher/dashboard/blob/53544f5d190ba53dd7b6c819208732efe30c2ee1/shell/config/labels-annotations.js#L113

Reply:
Appreciate it - fixed this now.